tests/Oryx.RuntimeImage.Tests/PhpImageTest.cs

```diff
@@ -64,5 +65,43 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
             Assert.True((bool)((JValue)gdInfo.GetValue("JPEG Support")).Value);
             Assert.True((bool)((JValue)gdInfo.GetValue("PNG Support")).Value);
         }
+
+        [SkippableTheory]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        [InlineData("5.6")]
+        public void PhpRuntimeImage_Contains_VersionAndCommit_Information(string version)
+        {
+            var agentOS = Environment.GetEnvironmentVariable("AGENT_OS");
+            var gitCommitID = Environment.GetEnvironmentVariable("BUILD_SOURCEVERSION");
+            var buildNumber = Environment.GetEnvironmentVariable("BUILD_BUILDNUMBER");
+            var expectedOryxVersion = string.Concat(Settings.OryxVersion, buildNumber);
+
+            // we cant always rely on gitcommitid as env variable in case build context is not correctly passed
+            // so we should check agent_os environment variable to know if the build is happening in azure devops agent
+            // or locally, locally we need to skip this test
+            Skip.If(string.IsNullOrEmpty(agentOS));
+
+            // Act
+            var result = _dockerCli.Run(new DockerRunArguments
+            {
+                ImageId = "oryxdevms/php-" + version + ":latest",
+                CommandToExecuteOnRun = "oryx",
+                CommandArguments = new[] { " " }
```

Comment:
need not give anything? doesn't it work without setting this property?

Reply:
it was failing for me .. so this is quick fix .. I am working on it seperately